app/assets/js/generator.js

```diff
@@ -63,7 +63,7 @@
 
         var list = new List('plugins-all', {
           valueNames: [
-            'name',
+            'name-website',
```

Comment:
List.js checks the inner content for sorting.
`<span class="name-website">` is more specific than `<td class="name">` and this improves the performance.